src/lib/util.ts

```diff
@@ -231,6 +231,18 @@ export function parseToNumberString(num: number, getString: GetStringFunction):
 	return `${num}`
 }
 
+export function discordLocaleToBotLocale(discordLocale: string): string {
+	if (discordLocale === "en-US" || discordLocale === "en-GB") discordLocale = "en"
+	else if (discordLocale === "es-ES") discordLocale = "es"
+	else if (discordLocale === "sv-SE") discordLocale = "sv"
+	else if (discordLocale === "uk") discordLocale = "ua"
+
+	discordLocale = discordLocale.replace(/([a-z]{2})(?:-([A-Z]{2}))?/, (_, a, b) => (b ? a + b.toLowerCase() : a))
+
+	if (readdirSync("./strings").includes(discordLocale)) return discordLocale
```

Comment:
```suggestion
	if (discordLocale === "uk") discordLocale = "ua"

	const localeWithCountryCode = discordLocale.replace(/([a-z]+)(?:-([A-Z]+))?/, (_, a, b) => (b ? a + b.toLowerCase() : a))
	const locale = discordLocale.replace(/([a-z]+)?/, (_, a) => a)

	if (readdirSync("./strings").includes(localeWithCountryCode)) return localeWithCountryCode
	else if (readdirSync("./strings").includes(locale)) return locale
```

Reply:
Like the idea. Implemented in the latest commit with a few minor tweaks.